client/src/components/molecules/Hero.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from "react";
+import Button from "../atoms/Button";
+import { Icon } from "../atoms/Icon/Icon";
+import { ICON_NAMES } from "../atoms/Icon/IconNames";
+
+import TextHeadline from "../atoms/TextHeadline";
+import TextBody from "../atoms/TextBody";
+import DividerWithText from "../atoms/DividerWithText";
+
+interface Props {
+  loginStatus: boolean;
+}
+export default function Hero({ loginStatus }: Props) {
+  //need to lift this state
+  const [popUp, displayPopUp] = useState(false);
+  return (
+    //hero container
+    <>
+      <div className="flex flex-col items-center">
+        <div className="w-full max-w-sm xs:max-w-xs">
+          {(!loginStatus && <Icon name={ICON_NAMES.hero_image} viewBox={"0 0 470 325.475"} className="w-full h-auto" />) || <Icon name={ICON_NAMES.profile_welcome} viewBox={"0 0 153.993 159.24"} className="w-full h-auto" />}
+        </div>
+        <TextHeadline size="sm" variant="h1" className="pt-6 pb-8">
+          {(!loginStatus && "A place where musicians find other musicians and play music together") || `Welcome back '${"user.name"}'!`}
+        </TextHeadline>
+        <div className="flex flex-row items-center justify-between gap-3 w-full items-stretch text-blue-500">
+          <Button
+            buttonState="default"
+            buttonLabel="Find posts"
+            buttonVariant="secondary"
+            iconPosition="top"
+            icon={ICON_NAMES.author_icon}
+            iconHeight={13.887}
+            iconWidth={13.887}
+            iconViewbox={"0 0 13.887 13.887"}
+            onClick={() => !loginStatus && displayPopUp(true)}
+            to={loginStatus ? "/posts" : undefined}
+            className="no-underline w-full"
+          ></Button>
+          <Button
+            buttonState="default"
```

Comment:
Not a big thing and maybe not even for this pr but I think we will need to remove the state from the button to be a mandatory prop as it is not needed only in special cases I guess

Reply:
Agree with this, we can add it in the next branch/feature